src/bee-module.nix

```diff
@@ -111,14 +119,23 @@
         inherit (evaled.config.bee) system pkgs;
         inherit (evaled.config.bee.pkgs) config; # nixos modules don't load this
       };
-      # seemlessly integrate hm if desired
-      imports = l.optionals evaled.options.bee.home.isDefined [
-        evaled.config.bee.home.nixosModules.home-manager
-        {
-          home-manager.useGlobalPkgs = true;
-          home-manager.useUserPackages = true;
-        }
-      ];
+
+      imports =
+        # seemlessly integrate hm if desired
+        l.optionals evaled.options.bee.home.isDefined [
+          evaled.config.bee.home.nixosModules.home-manager
+          {
+            home-manager.useGlobalPkgs = true;
+            home-manager.useUserPackages = true;
+          }
+        ]
+        # seemlessly integrate nixos-wsl if desired
+        ++ l.optionals evaled.options.bee.wsl.isDefined [
+          evaled.config.bee.wsl.nixosModules.wsl
+          {
+            wsl.enable = l.mkDefault true;
+          }
+        ];
```

Comment:
Config/integration-wise: is that all it takes?

Reply:
According to the official [NixOS-WSL configuration.nix example](https://github.com/nix-community/NixOS-WSL/blob/main/configuration.nix), this should be enough. 

The attrs
```
wsl.wslConf.automount.root = "/mnt";
wsl.defaultUser = "nixos";
```

are set to default values from module ([defaultUser here](https://github.com/nix-community/NixOS-WSL/blob/main/modules/wsl-distro.nix#L27) and [wslConf.automount.root here](https://github.com/nix-community/NixOS-WSL/blob/main/modules/wsl-conf.nix#L22)) in provided example configuration

Reply:
Ok, sounds good, thanks for double checking!